### PR TITLE
feat(operator): Add cli to query opt info

### PR DESCRIPTION
**Pull request by TimmyExogenous**
## Description

This PR adds a command-line command to query the opted info for a specified operator and AVS.

### CODE REVIEW

File: x/operator/client/cli/query.go [hunk 1]
```diff
@@ -415,3 +417,39 @@ func QueryAllAVSsByOperator() *cobra.Command {
...
+			}
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			queryClient := operatortypes.NewQueryClient(clientCtx)
+			req := &operatortypes.QueryOptInfoRequest{
+				Details: &operatortypes.OperatorAVSAddressDetails{
+					OperatorAddr: args[0],
+					AVSAddress:   args[1],
```
Comment (MaxMustermann2) [inline x/operator/client/cli/query.go]: Please validate it to be a hex address and convert it to the expected lower/upper/checksum case

File: proto/exocore/operator/v1/query.proto [hunk 2]
```diff
@@ -294,4 +302,11 @@ service Query {
       get: "/exocore/operator/v1/opt/avs_list/{operator}"
     };
   }
+
+  // QueryOptInfo queries specified opted information.
+  rpc QueryOptInfo(QueryOptInfoRequest) returns (OptedInfo) {
+    option (google.api.http) = {
+      get: "/exocore/operator/v1/opt_info"
```
Comment (MaxMustermann2) [inline proto/exocore/operator/v1/query.proto]: Can this URL include the params like the one immediately above this one?

**Review by MaxMustermann2 — APPROVED** [review]

**Review by leonz789 — APPROVED** [review]

File: x/operator/client/cli/query.go (hunk 1 above)
Comment (TimmyExogenous) [reply x/operator/client/cli/query.go]: Thanks, It has been fixed.

File: proto/exocore/operator/v1/query.proto (hunk 2 above)
Comment (TimmyExogenous) [reply proto/exocore/operator/v1/query.proto]: Thanks, It has been fixed.

File: x/operator/client/cli/query.go [hunk 3]
```diff
@@ -314,15 +322,19 @@ func QueryAVSUSDValue() *cobra.Command {
...
 func QueryOperatorSlashInfo() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "QueryOperatorSlashInfo <operatorAddr> <avsAddr>",
-		Short: "Get the the slash information for the operator",
-		Long:  "Get the the slash information for the operator",
-		Args:  cobra.ExactArgs(2),
+		Use:     "QueryOperatorSlashInfo <operatorAddr> <avsAddr>",
+		Short:   "Get the the slash information for the operator",
+		Long:    "Get the the slash information for the operator",
+		Example: "exocored query operator QueryOperatorSlashInfo  exo1c5x7mxphvgavjhu0au9jjqnfqcyspevtyy27mz 0xaa089ba103f765fcea44808bd3d4073523254c57",
```
Comment (MaxMustermann2) [inline x/operator/client/cli/query.go]: nit: extra space
```suggestion
		Example: "exocored query operator QueryOperatorSlashInfo exo1c5x7mxphvgavjhu0au9jjqnfqcyspevtyy27mz 0xaa089ba103f765fcea44808bd3d4073523254c57",
```

**Review by MaxMustermann2 — APPROVED** [review]
Thanks, the `embed` proto option is great!

File: x/operator/client/cli/query.go (hunk 3 above)
Comment (TimmyExogenous) [reply x/operator/client/cli/query.go]: Thanks, fixed.

**Review by trestinlsd — APPROVED** [review]

**Pull request merged**